docs/content_management/taxonomy/taxonomy.md

```diff
@@ -36,6 +36,17 @@ To do it, first, create a new Content Type with `content_category` identifier an
 
 Next, in `config/packages/ibexa_taxonomy.yaml` add the following configuration:
 
+``` yaml
+[[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml', 1,16 )=]]
+```
+
+### Hide Content tab
+
+The **Content** tab in taxonomy objects, for example, tags and categories, lists all Content assigned to the current taxonomy. 
+You can hide the **Content** tab in the **Categories** view.
+
+In the `config/packages/ibexa_taxonomy.yaml` add `assigned_content_tab` with the flag `false` (for other taxonomies this flag is by default set to `true`):
+
 ``` yaml
 [[= include_file('code_samples/taxonomy/config/packages/ibexa_taxonomy.yaml')=]]
```

Comment:
Select a range of lines here, maybe? Now it pretty much repeats the code block above